packages/sysutils/busybox/scripts/getedid

```diff
@@ -0,0 +1,274 @@
+      rm "$file"
+      mv "$file".old "$file"
+      mount_ro
+      rm -fr /storage/cpio
+      rm -fr /storage/.config/firmware
+      sys_reboot
+    else
+      echo "You don't have a backup file for $file. You did not use this script to create the custom EDID"
+      echo "Therefore we can't be sure the script is working prooerly. Exiting"
+      exit 3
+    fi
+    rm -f /storage/.config/xorg.conf
+    rm -f /storage/.config/xorg.le.backup
+    sys_reboot
+  fi   
+}
+
+
+# run main script depending on the GPU
+run() {
+  if [ "$gpu" = "intel" ]; then
+    intel 
+  elif [ "$gpu" = "nvidia" ]; then
+    nvidia
+  fi
+}
+
+
+# mounting /flash to rw
+mount_rw() {
+  mount -o remout,rw /flash
+}
+
+
+# mounting /flash to ro
+mount_ro() {
+  mount -o remount,ro /flash
+}
+
+
+# remount /flash to ro and reboot
+sys_reboot() {
+  mount_ro
+  echo "The system is rebooting in 15 seconds"
+  sleep 15
+  reboot
+}
+
+
+# check syslinux.cfg and/or extlinux.conf
+check_file() {
+  # check boot system
+  if [ -d /sys/firmware/efi]; then
+    boot="UEFI"
+    sys_path="flash/EFI/BOOT"
+  else
+    boot="BIOS"
+    sys_path="/flash"
+  fi
+
+  #check which file is available
+  if [ -f "$sys_path/syslinux.cfg" ] && [ -f "$sys_path/extlinux.conf" ];then
+    echo "Your system contains both. A /flash/syslinux.cfg and a /flash/extlinux.conf file"
+    echo "Something is wrong on your system. Exiting"
+    exit 4
+  elif [ -f "$sys_path/extlinux.conf" ];then
+    file="$sys_path/extlinux.conf"
+  elif [ -f "$sys_path/syslinux.cfg" ];then
+    file="$sys_path/syslinux.cfg"
+  else
+    echo "You neither have a extlinux.conf nor do you have a syslinux.cfg."
+    echo "Dual boot systems aren't supported"
+    exit 2
+  fi
+}
+
+
+check_content() {
+  # check if changes are already made to $file and exit if yes
+  if [ "$(grep "APPEND" $file)" != " APPEND boot=LABEL=System disk=LABEL=Storage ssh quiet" ]; then
+    echo "You alread did some changes to $file. Exiting."
+    exit 5
+  fi
+}
+
+
+# restore the backup done by the script and depending on the GPU
+restore() {
+  if [ "$gpu" = "intel" ]; then
+    check_file
+    if [ -f "$file".old ]; then
+      mount_rw
+      cp "$file".old "$file"  # cp because of keeping the backup...just in case
+    else
+      echo "You didn't created a custom EDID yet or you did not use this script for this task."
+      echo "Therefore we can't ensure the is script working properly."
+      echo "Exiting"
+      exit 3
+    fi
+  elif [ "$gpu" = "nvidia" ]; then
+    if [ -f /storage/.config/xorg.conf ]; then
+      mv /storage/.config/xorg.conf /storage/.config/xorg.le.backup
+    else
+      echo "You don't have a xorg.conf in your .config folder. Exiting"
+      exit 7
+    fi
+  fi
+  sys_reboot
+}
+
+
+intel() {
+  #check which output is connnected:
+  for i in /sys/class/drm/*; do
+    if [ "$(cat "$i"/status 2>/dev/null)" = "connected" ]; then
+      hdmi="$(echo "$i" | cut -d / -f 5 | sed 's/card[0-9]-//g')"
```

Comment:
Also, you need to keep the `card0-` prefix, at least on my Skylake, in which case you just need:
```
      hdmi="$(basename "$i")"
```